problems/4.arrays.js

```diff
@@ -2,25 +2,44 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+function firstAndLastAverage(arr) {
+  if(arr.length>=2){
```

Comment:
I'd start this as:
if(arr.length < 2) return null; 
And then the rest of the code.